src/Prometheus/Storage/APC.php

```diff
@@ -162,6 +174,9 @@ public function updateCounter(array $data): void
             $old = apcu_fetch($valueKey);
             if ($old !== false) {
                 $done = apcu_cas($valueKey, $old, $this->toBinaryRepresentationAsInteger($this->fromBinaryRepresentationAsInteger($old) + $data['value']));
+            } else {
+                apcu_add($this->valueKey($data), 0);
```

Comment:
Is there any reason why you don't check if it is $new here like on the other places?

Reply:
Sorry i totally missed your comment!